engine/src/main/java/org/terasology/engine/bootstrap/EntitySystemSetupUtil.java

```diff
@@ -148,16 +139,16 @@ public static void addEntityManagementRelatedClasses(Context context) {
     }
 
     private static EventSystem createEventSystem(NetworkSystem networkSystem, PojoEntityManager entityManager, EntitySystemLibrary library,
-                                                 RecordedEventStore recordedEventStore, EntityIdMap entityIdMap,
-                                                 RecordAndReplaySerializer recordAndReplaySerializer, RecordAndReplayUtils recordAndReplayUtils) {
+                                                 RecordedEventStore recordedEventStore, RecordAndReplaySerializer recordAndReplaySerializer,
```

Comment:
Low priority / subjective: would it be worth making some sort of wrapper here? When creating an event system 4 out of 7 parameters are related to R&R, that seems like a lot :-)